src/components/access_group/AccessGroupMemberItem.vue

```diff
@@ -0,0 +1,180 @@
+<template>
+  <div
+    :class="{ 'list-item': true, primary: isCurator, expanded: showExpandable }"
+  >
+    <article class="list-item__main">
+      <div class="member-view">
+        <UserPicture
+          :avatar="{ picture: member.picture, username: member.name }"
+          class="member-view__image"
+          :size="40"
+          :isStaff="true"
+        />
+        <div class="member-view__info">
+          <p class="info-header">{{ member.name }}</p>
+          <p class="info-sub">
+            <Icon v-if="isCurator" id="crown-fill" :width="16" :height="16" />
+            {{ member.role }}
+          </p>
+        </div>
+      </div>
+      <button
+        :class="{ 'member-action': true, expanded: showExpandable }"
+        v-on:click="toggleExpandable"
+      >
+        <Icon id="info" :width="24" :height="24" />
+      </button>
+    </article>
+    <aside class="list-item__expandable" v-if="showExpandable">
+      <p class="expandable-row">
+        <span class="expandable-row__label">{{
+          fluent('access-group_members', 'member-expandable_member-since')
+        }}</span>
+        <span class="expandable-row__content">
+          {{ formatDate(member.since) }}
+        </span>
+      </p>
+      <p class="expandable-row" v-if="!member.added_by.isAnonymous()">
+        <span class="expandable-row__label">{{
+          fluent('access-group_members', 'member-expandable_added-by')
+        }}</span>
+        <span class="expandable-row__content">{{ member.added_by.name }}</span>
+      </p>
+    </aside>
+  </div>
+</template>
+
+<script>
+import UserPicture from '@/components/ui/UserPicture.vue';
+import EditButton from '@/components/ui/EditButton.vue';
+import Icon from '@/components/ui/Icon.vue';
+import {
+  DISPLAY_MEMBER_ROLES,
+  MEMBER_IDEX,
+} from '@/view_models/AccessGroupViewModel';
+
+export default {
+  name: 'AccessGroupMemberItem',
+  components: { UserPicture, EditButton, Icon },
+  props: {
+    member: Object,
+  },
+  methods: {
+    toggleExpandable() {
+      this.showExpandable = !this.showExpandable;
+    },
+    // TODO: This needs to be done in the vuex model
+    formatDate(dateString) {
+      return new Date(dateString).toLocaleDateString('en-US');
+    },
+  },
+  computed: {
+    addedByProfile() {
```

Comment:
not used and `addedByMemberId` doesn't exist afaik. ( /p/ also needs a username)